minecraft/mappings_old/fabric-1.20.4-1.20.1.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
wha

Reply:
we should likely just remove that whole dirsctory

Reply:
Yes